azurerm/internal/services/keyvault/key_vault_certificate_resource.go

```diff
@@ -620,7 +625,8 @@ func expandKeyVaultCertificatePolicy(d *schema.ResourceData) keyvault.Certificat
 	issuers := policyRaw["issuer_parameters"].([]interface{})
 	issuer := issuers[0].(map[string]interface{})
 	policy.IssuerParameters = &keyvault.IssuerParameters{
-		Name: utils.String(issuer["name"].(string)),
+		Name:            utils.String(issuer["name"].(string)),
+		CertificateType: utils.String(issuer["certificate_type"].(string)),
```

Comment:
this'll be passed as an empty string if unset.. does this need to be conditionally set?